src/components/swap/SwapModalHeader.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { sendAnalyticsEvent } from '@uniswap/analytics'
+import {
+  testAllowedSlippage,
+  testRecipientAddress,
+  testTradeExactInput,
+  testTradeExactOutput,
+} from 'test-utils/constants'
+import { render, screen, within } from 'test-utils/render'
+import noop from 'utils/noop'
+
+import SwapModalHeader from './SwapModalHeader'
+
+jest.mock('@web3-react/core', () => {
+  const web3React = jest.requireActual('@web3-react/core')
+  return {
+    ...web3React,
+    useWeb3React: () => ({
+      chainId: 1,
+    }),
+  }
+})
+
+jest.mock('@uniswap/analytics')
+const mockSendAnalyticsEvent = sendAnalyticsEvent as jest.MockedFunction<typeof sendAnalyticsEvent>
+
+describe('SwapModalHeader.tsx', () => {
+  let sendAnalyticsEventMock: jest.Mock<any, any>
+
+  beforeAll(() => {
+    sendAnalyticsEventMock = jest.fn()
+  })
+
+  it('matches base snapshot, test trade exact input', () => {
+    const { asFragment } = render(
+      <SwapModalHeader
+        trade={testTradeExactInput}
+        allowedSlippage={testAllowedSlippage}
+        shouldLogModalCloseEvent={false}
+        showAcceptChanges={false}
+        setShouldLogModalCloseEvent={noop}
+        onAcceptChanges={noop}
+        recipient={testRecipientAddress}
+      />
+    )
+    expect(asFragment()).toMatchSnapshot()
+    expect(screen.getByText(/Output is estimated. You will receive at least /i)).toBeInTheDocument()
+    expect(screen.getByTestId('input-symbol')).toHaveTextContent('ABC')
```

Comment:
Would it make more sense to use values from the input props here?

```suggestion
    expect(screen.getByTestId('input-symbol')).toHaveTextContent(testTradeExactInput. inputAmount.currency.symbol)
```